index.html

```diff
@@ -0,0 +1,160 @@
+          <p>
+            Solange Piaget Knowles is an American singer, songwriter, record producer, performance artist, and actress. 
+            Expressing an interest in music from an early age, Knowles had several temporary stints as a backup dancer 
+            for Destiny&apos;s Child, which featured her elder sister, Beyoncé Knowles, among its members, before signing 
+            with her father Mathew Knowles&apos;s Music World Entertainment label. At age 16, Knowles released her first 
+            studio album Solo Star (2002). Between 2005 and 2007, Knowles had several minor acting roles, including the 
+            direct-to-video Bring It On: All or Nothing (2006) and continued writing music for Beyoncé and for former 
+            Destiny&apos;s Child group members Kelly Rowland and Michelle Williams.
+          </p></li>
+  
+  <form> 
+    <label for="purchase tickets page"> PURCHASE TICKETS</label><br>
+    <p>
+      Each ticket comes with one full day of admission. The cost of each ticket is the same no matter the day of attendance, 
+      or events scheduled that day. <strong>DISCLAIMER:</strong> One ticket will only grant you one day of admission. If you wish to attend multiple dates, 
+      you must purchase multiple tickets. <strong>Each ticket costs $50</strong> and covers one person for a full day of activities. 50% of the proceeds from 
+      ticket sales will go to the Global Fund for Women charity, supporting women worldwide and their pursuit of education and equality. 
+      Learn more at the <a href="https://join.globalfundforwomen.org/a/cye2020a?ms=adwords&gclid=CjwKCAiAq8f-BRBtEiwAGr3DgQYd4C5Lyx_T3hkBVewJ3XnAL8c8J1MqMhgL7cvQHtkBCv4bOoDy1RoC46gQAvD_BwE">
+      Global Fund for Women</a> website.
+    </p>
+    <label for="attending"> SELECT DATES BELOW</label><br>
+    <input type="checkbox" id="thursday"> <label>Thursday May 20th 2021</label><br>
+    <input type="checkbox" id="friday"><label>Friday May 21st 2021</label><br>
+    <input type="checkbox" id="saturday"><label>Saturday May 22nd 2021</label><br>
+    <label>CHECKOUT</label><br>
+    <label>NAME</label><br> 
+    <input type="text" id="name"><br>
+    <label>ADDRESS</label><br> 
+    <input type="text" id="address"><br>
+    <label>CITY</label><br> 
+    <input type="text" id="city"><br>
+    <label>STATE</label><br> 
+    <input type="text" id="state"><br>
+    <label>ZIPCODE</label><br> 
+    <input type="text" id="zipcode"><br>
+    <label>CARD NUMBER</label><br> 
+    <input type="text" id="card number"><br>
+    <label>EXP DATE</label><br> 
+    <input type="text" id="exp date"><br>
+    <label>CVV</label><br> 
+    <input type="text" id="cvv"><br>
+    <input type="submit" id="confirm">
+    </p>
+  </form> 
+  <button type="button">
+  <img src="https://cdn.glitch.com/880db4f1-e059-4f38-b845-ad2307b0e837%2FCOUNT%20ME%20IN%20BUTTON-01.png?v=1607638815064" alt="plum colored button reading 'count me in'">
+  </button>
```

Comment:
Can you tell me more about what this button should do?

Reply:
This button will take you to the "purchase tickets" portion of the page.

Reply:
If it is linking to another part of this page or a separate page, it should be a link (`a`) and not a `button` -- also, note that even if it's a link element, it can still be styled to look like a button (e.g., background color, borders, rounded corners, whatever); the look of the thing and the markup are separate 👍🏻